assets/performanceprofile/configs/99-runtimes.conf

```diff
@@ -4,11 +4,6 @@ infra_ctr_cpuset = "{{.ReservedCpus}}"
 {{if .SharedCpus}}shared_cpuset = "{{.SharedCpus}}"{{end}}
 {{end}}
 
-# We should copy paste the default runtime because this snippet will override the whole runtimes section
```

Comment:
Why is this logic unnecessary?

Reply:
@rbaturov Please add test to verify that.
You can simply list crio config as part of the test and see that everything is in place

Reply:
this should also be documented in the commit message of the related change:
"this logic is unnecessary _because_..."

Reply:
@ffromani Defentilty will put a more detailed message for this commit (This was just a draft)